File: practice_java_program/notpad/ShiftZeroRightArray.java
public class ShiftZeroRightArray{
	public static void main(String[] args){
		int [] numbers = {4,0,3,0,5,0,6,0};
		int index=0;
		for(int i=0; i<numbers.length; i++){
			if(numbers[i] !=0){
				numbers[index]=numbers[i];
				index++;
			}
		}
		while(index<numbers.length){
			numbers[index]=0;
			index++;
		}
		for(int num:numbers){
			System.out.print(num+", ");
		}
	}
}

File: python_oriented_interview/notpad/ShiftZeroRightArray.py
numbers = [4, 0, 3, 0, 5, 0, 6, 0]
index = 0
for i in range(len(numbers)):
    if numbers[i] != 0:
        numbers[index] = numbers[i]
        index += 1

while index < len(numbers):
    numbers[index] = 0
    index += 1

for num in numbers:
    print(num, end=", ")
print()
